Assets/AppCenter/Editor/AppCenterPostBuild.cs

```diff
@@ -419,7 +420,21 @@ private static void OnPostprocessInfo(PlistDocumentWrapper info, AppCenterSettin
     {
         if (settings.UseDistribute && AppCenter.Distribute != null)
         {
-            // Add App Center URL sceme.
+            // Add App Center URL scemes.
+            var schemes = new List<string>() { "appcenter-" + settings.iOSAppSecret  };
+
+            // Create a reflection call for getting custom schemes from iOS settings.
+            Type playerSettingsClass = typeof(PlayerSettings.iOS);
+            MethodInfo iOSURLSchemesMethod = playerSettingsClass.GetMethod("GetURLSchemes", BindingFlags.Static | BindingFlags.NonPublic);
```

Comment:
Do we know what versions of Unity Editor have this one?

Reply:
I tested these changes on LTS versions of Unity 2018, 2019 and 2020. All works correctly.